compat/mingw.c

```diff
@@ -3304,13 +3304,8 @@ const char *program_data_config(void)
 
 	if (!initialized) {
 		const char *env = mingw_getenv("PROGRAMDATA");
-		const char *extra = "";
-		if (!env) {
-			env = mingw_getenv("ALLUSERSPROFILE");
-			extra = "/Application Data";
-		}
```

Comment:
I admit, my only intention was to avoid dragging around the outdated description.